### PR TITLE
Fix minio http/https issue & refactor notebooks

### PR DESCRIPTION
**Pull request by rohank07**
Add `creds['MINIO_URL'].startswith('https')` to handle https minio url

### CODE REVIEW

File: self-serve-storage/R/daaas_storage.R [hunk 1]
```diff
@@ -65,7 +65,8 @@ daaas_storage.__getClient__ <- function (storage_type) {
         "AWS_S3_ENDPOINT" =  ENDPOINT,
         "AWS_ACCESS_KEY_ID" = MINIO_ACCESS_KEY,
         "AWS_SECRET_ACCESS_KEY" = MINIO_SECRET_KEY,
-        "AWS_DEFAULT_REGION" = ""
+        "AWS_DEFAULT_REGION" = "",
```
Comment (Jose-Matsuda) [inline self-serve-storage/R/daaas_storage.R:68]: Just a reminder to apply marberts suggestion here if needed.

**Review by Jose-Matsuda — CHANGES_REQUESTED** [review]
few small things and just a reminder to apply marberts suggestion to the one file as well.

File: querySQL/daaas_storage_boto3.py [hunk 2]
```diff
@@ -54,12 +54,13 @@ def __get_minio_client__(tenant):
         p = subprocess.Popen(CMD, stdout=subprocess.PIPE, shell=True,
                              executable='/bin/bash')
         d[var] = p.stdout.readlines()[0].strip().decode("utf-8")
-
+    HOST = d['MINIO_URL']
```
[the remark this answers is not in the document]
Comment (rohank07) [reply querySQL/daaas_storage_boto3.py]: Removed the extra variable.

**Review by Jose-Matsuda — APPROVED** [review]
lgtm

**Pull request merged**